Readme.md

```diff
@@ -1,82 +1,3 @@
-# Today's Objective: Live CSS Editor
+# https://codepen.io/hmr/pen/MWwMaxO
```

Comment:
I don't see any input field unless something is wrong with my Chrome browser.

https://www.dropbox.com/s/38lnm0vp711yu9f/Screenshot%202020-04-04%2015.40.18.png?dl=0

Reply:
it's because I didn't figure out how to use ace with codepen. thanks for trying to look, though!